flake8_spellcheck/python.txt

```diff
@@ -95,6 +95,7 @@ mypy
 namedtuple
 nargs
 netloc
+noqa
```

Comment:
this isn't currently necessary for this pr, but it was in a previous iteration. i still think it should be in the dictionary, but i can remove it if requested